code/game/objects/items/weapons/implants/implant_loyality.dm

```diff
@@ -11,7 +11,7 @@
 				<b>Important Notes:</b> Personnel injected with this device tend to be much more loyal to the company.<BR>
 				<HR>
 				<b>Implant Details:</b><BR>
-				<b>Function:</b> Contains a small pod of nanobots that manipulate the host's mental functions.<BR>
+				<b>Function:</b> Contains a small pod of nanobots that will protect the hosts mental functions from outside influences.<BR>
```

Comment:
pretty sure the correct form would be "host's" as it's a possessive. "hosts" is a plural which would refer to multiple hosts, which is not the goal of this sentence. :cop: grammar police :cop: